gradle.properties

```diff
@@ -8,7 +8,7 @@ pluginVersion=0.4.2
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
 pluginSinceBuild=223
-pluginUntilBuild=999.*
+pluginUntilBuild=243.*
```

Comment:
Up until this point this has been a warning, not an error. Updated it to match the latest version of IDEA however this means in 2025 we'll have to re-release a new version of the plugin after we've tested with the latest release.